spoc/io.py

```diff
@@ -4,7 +4,10 @@
 from typing import Dict, Union
 import pandas as pd
 import dask.dataframe as dd
-from .models import FragmentSchema, AnnotatedFragmentSchema, HigherOrderContactSchema
+from .contacts import Contacts
+from .pixels import Pixels
+from .dataframe_models import FragmentSchema, ContactSchema
+from .fragments import Fragments
 
 
 class FileManager:
```

Comment:
I am not sure about wrapping IO functions in a class. It does encapsulate common behaviour, but this is not yet implemented.

Reply:
Okay, got it! Sounds practical :)